index.html

```diff
@@ -1,16 +1,59 @@
-<!DOCTYPE html>
-<html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Responsive Cake webpage</title>
-    <!-- Add a link to your css file here -->
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<meta http-equiv="X-UA-Compatible" content="ie=edge">
+<title>Responsive Cake webpage</title>
+<link rel="stylesheet" href="./style.css"></head>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+<!-- Add your markup here -->
+<header>
+<section class= "image-container" >
+<div class= "logo">
+    <img src="image/logo.jpg" alt="">
+</div>
+<div class= "textMenu-container" >
+    <img class="menu-icon" src="image/menu-hamburger.svg" alt=""/>
+    <section class="text">
+    <p>Best cake in town </p>
+    <p>Deliver to your door</p>
+</section>
+</div>
+</section>
+ <div class= "hamburger-menu">
+    <div><a class="hamburger-item" href="#Aboutus"><p>About us</p></a></div>
+    <div><a class="hamburger-item" href="#CustomerCare"><p>Customer Care</p></a></div>
```

Comment:
The `<p>` tag in your links is not necessary. You can simplify the link and just put
`<a class="hamburger-item" href="#Aboutus">About us</a>`